packages/wallet/api/src/sharingUnserializer.js

```diff
@@ -0,0 +1,72 @@
+// @ts-check
+import { Far, makeMarshal } from '@endo/marshal';
+
+/**
+ * Make a pair of unserializers that share low-privilege values.
+ *
+ * @param {string} [lowProperty]
+ */
+export const makeSharingUnserializer = (lowProperty = 'boardId') => {
```

Comment:
while this may be the common use case it doesn't make sense to me as a _default_.
```suggestion
export const makeSharingUnserializer = (lowProperty) => {
```

Reply:
Sure, removed the default